packages/nodes-base/nodes/Note/Note.node.ts

```diff
@@ -0,0 +1,58 @@
+import { IExecuteFunctions } from 'n8n-core';
+import {
+	INodeExecutionData,
+	INodeType,
+	INodeTypeDescription,
+} from 'n8n-workflow';
+
+
+export class Note implements INodeType {
+	description: INodeTypeDescription = {
+		displayName: 'Note',
+		name: 'note',
+		icon: 'fa:sticky-note',
+		group: ['input'],
+		version: 1,
+		description: 'Leave notes and organize your flow. Supports markdown.',
+		defaults: {
+			name: 'Note',
+			color: '#FFD233',
+		},
+		inputs: ['main'],
+		outputs: ['main'],
+		properties: [
+			{
+				displayName: 'Content',
+				name: 'content',
+				type: 'string',
+				required: true,
+				typeOptions: {
+					rows: 4,
+				},
+				default: 'I am a heading. This is how you bold text and this is how you create an inline link.',
```

Comment:
you need to change this intro markdown to match designs
- bold first line
- add new line between first and second sentence
- add link to link

Reply:
Updated as suggested